### ISSUE TITLE
Alt-art information?

### ISSUE THREAD

**Issue by Smashman** [zaroth/netrunner-cards-json#123]
Perhaps cards with alt-arts could have some amount of information?

Depending on if we're expecting cards to receive more than a single alt-art across the lifespan of the game, it could be a simple boolean, or an integer detailing how many alt-arts exist.

**Comment by jdcowf** [comment]
This would allow deck building based on inventory. If I had 2 AA PolOp's but didn't have Democracy and Dogma my deck building app could still show me 2 PolOp's being available for construction.

If we're going down this road it may be worthwhile to add a field for full bleed artworks and acrylics versions of cards. This is likely less useful than the above situation since the full bleeds are currently only in packs, but could be beneficial to someone.

**Comment by NoahTheDuke** [comment]
If we wanted to track this, I'd want it to be normalized: Make a "table" of releases (Region 2014, Worlds 2018, etc), make a table of "alt" kind (acrylic, full-bleed, etc), make a join table of card-by-name to release to kind.

Having a canonical listing of every single alt in existence would be sweet. With the 600 dpi scans, we have nearly everything already, so linking the two would be very helpful for the community.

**Comment by Smashman** [comment]
I did some work on this, when I suggested it. I could dig it out and let you take a look.

**Comment by NoahTheDuke** [comment]
That would be killer, yes! Thank you so much.

**Comment by x3N1GM4x** [comment]
Yeah, there'd obv be some data structure work here and it might be something that feeds into the planned schema rebuild quite neatly, but either way, I'm 100% behind this as an idea. 👍

**Comment by plural** [comment]
We are definitely going to add this, but it will be in the second wave of NRDB v2 updates.
